### DISCUSSION TITLE
What are the main differences between Maude and Lambdapi?

### DISCUSSION THREAD

**Discussion by guialvares** [Deducteam/lambdapi#1130] [Q&A] [answered]
I was prototyping semantics of [Solidity programming language](https://github.com/SolidiKeY/solidity-semantics-maude) in Maude and I have read in this [master course](https://blanqui.gitlabpages.inria.fr/1-33.html) that both language have general form of pattern-matching. So, I am curious to know about good features of both.

**Comment by fblanqui** [comment, marked as answer]
Hi. You can find a precise description of Lambdapi pattern matching algorithm implemented by Gabriel Hondet in http://dx.doi.org/10.4230/LIPIcs.FSCD.2020.35 . In contrast to Maude, it does not implement matching modulo AC. On the other hand, it allows non-linear higher-order pattern-matching.
